text/0000-maturity-of-otel.md

```diff
@@ -0,0 +1,149 @@
+# Framework for defining the maturity of OpenTelemetry and its SIGs
+
+On 08 Mar 2023, the OpenTelemetry GC and TC held an OpenTelemetry Leadership summit, discussing various topics. One of the themes we discussed was establishing standard rules for describing the maturity of the OpenTelemetry project. This OTEP summarizes what was discussed there and is intended to have the wider community provide feedback.
+
+This OTEP builds on what was previously communicated by the project, especially on the following page: https://opentelemetry.io/docs/reference/specification/versioning-and-stability.
+
+The Collector’s [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
```

Comment:
Which single quote do we want to use? (I noticed that on line 11 the ASCII version is used "project's maturity").

```suggestion
The Collector's [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
```

Reply:
Good catch -- the doc was written using Google Docs in collaboration with @dyladan, with a few parts added on a regular code editor to fit the format of an OTEP. I unified the quotes in the latest commit.